geowave-store/src/main/java/mil/nga/giat/geowave/store/filter/SpatialQueryFilter.java

```diff
@@ -188,7 +193,7 @@ public void fromBinary(
 					fieldId));
 		}
 		buf.get(theRest);
-		queryGeometry = GeometryUtils.geometryFromBinary(geometryBinary);
+		queryGeometry = geometryInterner.intern(GeometryUtils.geometryFromBinary(geometryBinary));
```

Comment:
I should be able to add a test in there for this;  it's probably worth testing for regardless in case implementations change.  But it does have an effect:

Without it
https://travis-ci.org/ngageoint/geowave/builds/56052150

With it
https://travis-ci.org/ngageoint/geowave/builds/56170108